crates/libcontainer/src/container/container_start.rs

```diff
@@ -39,10 +40,9 @@ impl Container {
             bail!(err_msg);
         }
 
-        let spec = self
-            .spec()
+        let config = YoukiConfig::load(self.root.join("config.json"))
```

Comment:
Can we have only one place that specificity `config.json`, so it is easier to change in the future? Maybe in `config.rs`?

Reply:
Certainly. I have renamed it to `yconfig.json`.